.snippets/code/mrl/evm-tx.js

```diff
@@ -0,0 +1,50 @@
+async function batchApproveTransferEVMTx(moonbeamAPI) {
+  // Get Batch, IERC20, ITokenBridge contracts
+  const Batch = new ethers.utils.Interface(abi.Batch);
+  const WrappedFTM = new ethers.utils.Interface(abi.IERC20);
+  // https://github.com/wormhole-foundation/example-token-bridge-relayer/blob/main/evm/src/token-bridge-relayer/TokenBridgeRelayer.sol
+  const TokenRelayer = new ethers.Contract(
+    XLABS_RELAYER_ADDRESS,
```

Comment:
what is the xLabs relayer address?

Reply:
Moonbase Alpha: `0x9563a59c15842a6f322b10f69d1dd88b41f2e97b`
Moonbeam: `0xcafd2f0a35a4459fa40c0517e17e6fa2939441ca`